components/TicketsSection.js

```diff
@@ -29,9 +29,9 @@ function TicketsTable () {
   )
 }
 
-function TicketsRow ({ title, date }) {
+function TicketsRow ({ title, date, isFull }) {
   return (
-    <div className='tickets-row'>
+    <div className={`tickets-row ${isFull ? 'full' : ''}`}>
```

Comment:
@soraxtend just suggestion krub, this works too `tickets-row ${isFull && 'full'}`